site/public/js/autograding-status.js

```diff
@@ -42,10 +42,12 @@ function updateTable() {
                         new_row.insertCell().innerHTML = course_name[1];
                         new_row.insertCell().innerHTML = key2;
                         const int_box = new_row.insertCell();
+                        // eslint-disable-next-line eqeqeq
                         if (info.interactive != 0) {
```

Comment:
Similar to above, we can replace these only if confident that it cannot be `''` or `[]`, so probably safe to just leave these as disabled for now.